aws-redshift-eventsubscription/src/main/java/software/amazon/redshift/eventsubscription/Translator.java

```diff
@@ -212,21 +206,44 @@ private static software.amazon.awssdk.services.redshift.model.Tag translateToSdk
         return GSON.fromJson(GSON.toJson(tag), software.amazon.awssdk.services.redshift.model.Tag.class);
     }
 
-    public static List<software.amazon.awssdk.services.redshift.model.Tag> translateToSdkTags(List<Tag> tags) {
-        return tags == null ? null : tags
-                .stream()
-                .map(Translator::translateToSdkTag)
-                .collect(Collectors.toList());
+    public static List<software.amazon.awssdk.services.redshift.model.Tag> translateToSdkTags(List<Tag> modelTags) {
```

Comment:
have you combine refactoring of Null checks in this PR?

Reply:
yes